### PR TITLE
Add changes to UPGRADE file

### PR DESCRIPTION
**Pull request by mpysiak**
| Q               | A
| --------------- | -----
| Branch?         | 2.0
| Bug fix?        | no
| New feature?    | no
| Related tickets | n/a

### CODE REVIEW

File: UPGRADE-2.0.md [hunk 1]
```diff
@@ -14,9 +20,35 @@
...
+   And the routes configuration paths:
+   ```diff
+   sylius_refund:
+   -   resource: "@SyliusRefundPlugin/Resources/config/routing.yml"
+   +   resource: "@SyliusRefundPlugin/config/routing.yml"
+   ```
+
+   And the paths to assets and templates if you are using them.
+
+
```
Comment (GSadee) [inline UPGRADE-2.0.md]:
```suggestion

```

File: UPGRADE-2.0.md [hunk 2]
```diff
@@ -14,9 +20,35 @@
...
+   ```
+
+   And the routes configuration paths:
+   ```diff
+   sylius_refund:
+   -   resource: "@SyliusRefundPlugin/Resources/config/routing.yml"
+   +   resource: "@SyliusRefundPlugin/config/routing.yml"
+   ```
+
+   And the paths to assets and templates if you are using them.
```
Comment (GSadee) [inline UPGRADE-2.0.md]:
```suggestion
   Adjust the paths to assets and templates if you are using them.
```

File: UPGRADE-2.0.md [hunk 3]
```diff
@@ -137,3 +169,164 @@
...
+        }
+        ```
+
+    - `Sylius\RefundPlugin\Action\Admin\RefundUnitsAction`:
+        ```diff
+        public function __construct(
+        -       private readonly SessionInterface|RequestStack $requestStackOrSession,
+        -       private readonly RequestCommandCreatorInterface|RefundUnitsCommandCreatorInterface $commandCreator,
+        +       private RequestStack $requestStack,
+        +       private RequestCommandCreatorInterface $commandCreator
```
Comment (GSadee) [inline UPGRADE-2.0.md]:
```suggestion
        +       private RequestCommandCreatorInterface $commandCreator,
```

File: UPGRADE-2.0.md [hunk 4]
```diff
@@ -137,3 +169,164 @@
...
+       ```
+
+   - `Sylius\RefundPlugin\CommandHandler\RefundUnitsHandler`:
+       ```diff
+       public function __construct(
+       -    private EntityManagerInterface $entityManager,
+       +    private ObjectManager $entityManager,
+       ) {
+       }
+       ```
```
Comment (GSadee) [inline UPGRADE-2.0.md]: Sth has gone wrong here, because I don't see such a constructor for this handler 🤔

Comment (GSadee) [reply UPGRADE-2.0.md]: What is more, `RefundUnitsHandler` looks like it needs some fixes in 2.0

File: UPGRADE-2.0.md [hunk 5]
```diff
@@ -137,3 +169,164 @@
...
+       +       private LineItemFactoryInterface $lineItemFactory,
+       ) {
+       }
+       ```
+
+   - `Sylius\RefundPlugin\Creator\RefundCreator`:
+       ```diff
+       public function __construct(
+       -    private EntityManagerInterface $entityManager,
+       +    private ObjectManager $entityManager,
```
Comment (GSadee) [inline UPGRADE-2.0.md]: 😅 
```suggestion
       -    private ObjectManager $entityManager,
       +    private EntityManagerInterface $entityManager,
```

File: UPGRADE-2.0.md [hunk 6]
```diff
@@ -137,3 +169,164 @@
...
+   - `Sylius\RefundPlugin\Validator\RefundUnitsCommandValidator`:
+      ```diff
+      public function __construct(
+      -       private ?iterable $refundUnitsBelongingToOrderValidators = null,
+      +       private iterable $refundUnitsBelongingToOrderValidators,
+      ) {
+      }
+      ```   
+      
+1. The following classes where deprecated in 1.x and have been removed in 2.0:
```
Comment (GSadee) [inline UPGRADE-2.0.md]:
```suggestion
1. The following deprecated classes and interfaces have been removed in 2.0:
```

File: UPGRADE-2.0.md
Comment (GSadee) [inline UPGRADE-2.0.md:1]: I believe there are some other aliases/ids that has been replaced and can be added here:
https://github.com/Sylius/RefundPlugin/pull/454/files#diff-57a2d391e31e0d1c39e9a12a231e9e3cb7e2554b7faa11c80ab43b7ce889c631R139-R144

File: UPGRADE-2.0.md (hunk 4 above)
Comment (mpysiak) [reply UPGRADE-2.0.md]: Fixed

File: UPGRADE-2.0.md [hunk 7]
```diff
@@ -14,11 +20,39 @@
...
+
+   Adjust the paths to assets and templates if you are using them.
 
-1. Aliases introduced in RefundPlugin 1.6 have now become the primary service IDs in RefundPlugin 2.0. 
+1. No need to overwrite templates:  
+   Thanks to the use of Twig Hooks and the refactoring of templates, you no longer need to overwrite templates to use plugin features.
+
+1. The following routes have been removed:  
+    - `sylius_refund_order_credit_memos_list`
+    - `sylius_refund_plugin_shop_order_credit_memos_partial`
```
Comment (Rafikooo) [inline UPGRADE-2.0.md:53]: I can still see these routes in the 2.0 branch.

File: UPGRADE-2.0.md [hunk 8]
```diff
@@ -14,11 +20,39 @@
...
+   imports:
+   -   - { resource: "@SyliusRefundPlugin/Resources/config/config.yml" }
+   +   - { resource: '@SyliusRefundPlugin/config/config.yaml' }
+   ```
+
+   And the routes configuration paths:
+   ```diff
+   sylius_refund:
+   -   resource: "@SyliusRefundPlugin/Resources/config/routing.yml"
+   +   resource: "@SyliusRefundPlugin/config/routing.yml"
```
Comment (Wojdylak) [inline UPGRADE-2.0.md:43]: Maybe let's change the file extension to `yaml`.

**Review by Wojdylak — APPROVED** [review]

File: UPGRADE-2.0.md [hunk 9]
```diff
@@ -14,11 +20,39 @@
...
 
-1. Aliases introduced in RefundPlugin 1.6 have now become the primary service IDs in RefundPlugin 2.0. 
+1. No need to overwrite templates:  
+   Thanks to the use of Twig Hooks and the refactoring of templates, you no longer need to overwrite templates to use plugin features.
+
+1. The following routes have been removed:  
+    - `sylius_refund_order_credit_memos_list`
+    - `sylius_refund_plugin_shop_order_credit_memos_partial`
+
+1. Aliases introduced in RefundPlugin 1.6 have now become the primary service IDs in RefundPlugin 2.0.
```
Comment (Rafikooo) [inline UPGRADE-2.0.md:55]: Redundant, as the same note is mentioned below.

**Review by Rafikooo — APPROVED** [review]

File: UPGRADE-2.0.md (hunk 7 above)
Comment (mpysiak) [reply UPGRADE-2.0.md:53]: Changes in https://github.com/Sylius/RefundPlugin/pull/458

**Pull request merged**